.github/workflows/build-binaries.yml

```diff
@@ -40,33 +150,50 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10
+
+      - name: Set environment
+        run: |
+          node ./scripts/set-git-tag-env.js
+
       - name: Build arm
         run: |
-            docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-            docker run --rm -v $(pwd):/usr/src/app/ arm32v7/node:12 /bin/sh -c 'cd /usr/src/app/; npm install; npm run compile:armv7'
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+          docker run --rm -v $(pwd):/usr/src/app/ arm32v7/node:12 /bin/sh -c 'cd /usr/src/app/; npm install; npm run build; node ./scripts/copy-node-bindings-path.js; npm run compile:armv7;'
```

Comment:
Is the build step required for compilation? It adds a considerable amount of time to the arm compilation.